scripts/gas-refund-program/staking/2.0/MerkleRedeemHelperSePSP1.ts

```diff
@@ -0,0 +1,46 @@
+import { constructHttpClient } from "../../../../src/lib/utils/http-client";
+import { MerkleTreeData } from "../../types";
+
+const merkleTreeDataUrlByEpoch1: Record<number, string> = {
+  // since epoch 32 we started distributing refund in sePSP1 on mainnet
+  32: 'https://gateway.pinata.cloud/ipfs/QmfJzuZp5rU9KedFAJ6zUfst91axqqgCAw28zjmsmk8GFX?_gl=1*118tlu9*_ga*YmFhNzhiNjgtZmFlZS00YzQ2LWIyMGEtODhmM2E2MTU2NDMx*_ga_5RMPXG14TE*MTY4MDAyNzA2Ni4xLjEuMTY4MDAyNzM5NS40Mi4wLjA'
```

Comment:
this probably needs to be improved somehow, because this way have to modify and redeploy on each distribution

Reply:
I agree. Best solution would be point an stable ipns route pointing at latest distribution registry